exporter/datadogexporter/examples/collector.yaml

```diff
@@ -13,6 +13,55 @@ receivers:
       # The presence of the gRPC section enables the gRPC server on the default port (4317)
       grpc:
 
+  # k8s_cluster receiver scrapes valuable Kubernetes metrics.
+  k8s_cluster:
+    collection_interval: 10s
```

Comment:
This needs appropriate roles to be setup https://github.com/DataDog/opentelemetry-helm-charts/blob/main/charts/opentelemetry-demo-datadog/templates/roles.yaml